netorgsna.py

```diff
@@ -189,6 +189,13 @@ def get_hostgroups_to_update(diff) -> set:
                     hostgroup = re.search(r"'(.*?)'", k, re.DOTALL).group()
                     hostgroup = hostgroup.strip("'")
                     update_set.add(hostgroup)
+            if 'values_changed' in diff:
+                print(diff['values_changed'])
+                print(type(diff['values_changed']))
```

Comment:
lines 192-193 seem to be leftovers from debugging, if so please delete them before merging